src/app/company-dialog/company-dialog.component.spec.ts

```diff
@@ -1,45 +1,47 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+// import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Some special reason to not use `xdescribe` instead? When running test, we don't will have notice about that test suite, with `xdescribe` it will appear on log as _skipped_